modules/mri_violations/test/mri_violationsTest.php

```diff
@@ -52,7 +200,231 @@ function testMriProtocolCheckViolationsDoesPageLoad()
+        $this->webDriver->findElement(WebDriverBy::Name("Filename"))
+             ->sendKeys("assembly/test2/test2/mri/test2/test2.mnc");
+        $this->webDriver->findElement(WebDriverBy::Name("filter"))->click();
+        $bodyText = $this->webDriver->findElement(WebDriverBy::Xpath(
+             "//*[@id='violationsTable']/tbody/tr[1]/td[2]")
+         )->getText();
+        $this->assertEquals("[Test]PatientName", $bodyText);
+
+        //testing search by Description
+        $this->webDriver->findElement(WebDriverBy::Name("Description"))->sendKeys
+                ("Test Series Description");
+        $this->webDriver->findElement(WebDriverBy::Name("filter"))->click();
+        $bodyText = $this->webDriver->findElement(WebDriverBy::Xpath(
+             "//*[@id='violationsTable']/tbody/tr[1]/td[2]")
+         )->getText();
+        $this->assertEquals("[Test]PatientName", $bodyText);
+
+        //testing search by site
+        $siteElement =  $this->safeFindElement(WebDriverBy::Name("Site"));
+        $site = new WebDriverSelect($siteElement);
+        $site->selectByVisibleText("TESTinPSC");
+        $this->webDriver->findElement(WebDriverBy::Name("filter"))->click();
+        $bodyText = $this->webDriver->findElement(WebDriverBy::Xpath(
+             "//*[@id='violationsTable']/tbody/tr[1]/td[2]")
+         )->getText();
+        $this->assertEquals("[Test]PatientName", $bodyText);
+
+        //testing search by Description
+        $this->webDriver->findElement(WebDriverBy::Name("SeriesUID"))->sendKeys("5556");
+        $this->webDriver->findElement(WebDriverBy::Name("filter"))->click();
+        $bodyText = $this->webDriver->findElement(WebDriverBy::Xpath(
+             "//*[@id='violationsTable']/tbody/tr[1]/td[2]")
+         )->getText();
+        $this->assertEquals("[Test]PatientName", $bodyText);
+    }
+    /**
+     * Tests that,in the not resolved menu, change the Resolution status of the first row.
+     * Save it and check it. 
+     *
+     * @return void
+     */
+    function testNotResolvedSaveButton()
+    {
+
+        $this->safeGet($this->url . "/mri_violations/");
+        $resolutionElement =  $this->safeFindElement(WebDriverBy::Name
+               ("resolvable[c57b919a921eaa1a43bb5e0c44cd4226]"));
+        $resolution = new WebDriverSelect($resolutionElement);
+        $resolution->selectByVisibleText("Inserted");
+        $this->safeClick(WebDriverBy::Name("fire_away"));
+
+        $resolutionElement = $this->safeFindElement(WebDriverBy::Name
+               ("resolvable[c57b919a921eaa1a43bb5e0c44cd4226]"));
+        $resolution = new WebDriverSelect($resolutionElement);
+
+        $value = $resolution->getFirstSelectedOption()->getAttribute('value');
+        $this->assertEquals("inserted",$value);
+    }
+}
+?>
+                                                                                                                                                                                    428,1         Bot
```

Comment:
It looks like there's an extra line at the bottom that says "428,1         Bot". I don't think this is supposed to be here.